importer/dataset.go

```diff
@@ -116,6 +117,50 @@ func mapDataset(filePath string) (*model.Dataset, error) {
 	return dataset, nil
 }
 
+func mapHierarchies(wdaHierarchy *wda.HierarchySummary) []*model.GeographicHierarchySummary {
```

Comment:
Just need to add a comment to explain that we can get an array, a single item not in an array or neither and that's the reason we require all these conditional statements.